### DISCUSSION TITLE
Error while creating a new user with userType

### DISCUSSION THREAD

**Discussion by Abhilash-Kadu** [Sunbird-Lern/Community#151] [Installation] [answered]
API endpoint: [localhost:9000/v1/user/create](url)
Request:

> {
    "request": {
        "firstName": "Test",
        "lastName": "user",
        "channel": "sunbird",
        "userName": "testuser",
        "email":"testuser@test.com",
        "emailVerified": true,
        "password": "Test@123",
        "userType": "teacher"
    }
}

Response:

> {
    "id": "api.user.create",
    "ver": "v1",
    "ts": "2023-08-10 09:29:00:081+0530",
    "params": {
        "resmsgid": "bd66e439-3a58-48e8-8fc8-7dd132ee367c",
        "msgid": "bd66e439-3a58-48e8-8fc8-7dd132ee367c",
        "err": "UOS_USRCRTnull",
        "status": "FAILED",
        "errmsg": "userType config is empty for the statecode default"
    },
    "responseCode": "CLIENT_ERROR",
    "result": {}
}

**Comment by Jayaprakash8887** [comment, marked as answer]
@Abhilash-Kadu Please refer to following Discussion threads. Request you to check the "discussions" if your concern is answered in any of the other threads. If not, you are welcome to raise a new one. Please revert if your concern is not addressed in below threads. Kindly mark the thread as answered if your concern is resolved.
https://github.com/orgs/Sunbird-Lern/discussions/106
https://github.com/orgs/Sunbird-Lern/discussions/138
